src/election.html

```diff
@@ -16,7 +16,7 @@
   <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
 
   <script src="assets/irma.js" defer></script>
-  <script src="irma-functions.js" defer></script>
+  <script src="assets/js/irma-functions.js" defer></script>
```

Comment:
Mm... I think assets are like external stuff!

Reply:
Can be a way of facing it. 
I then should move css/ and apk/ out of assets folder, right?

Reply:
Deal!

It was good to bring the discussion and we can agree with that when the time comes. ;-)